cpp/ps0003/problem solve 3.cpp

```diff
@@ -1,12 +1,16 @@
 #include <iostream>
 using namespace std;
 int main(){
+    // prompting user for number 2/2
     int number;
     cout << "enter number \n";
     cin >> number;
-    if (number %2 ==0)
-    cout << "number is even \n";
+
```

Comment:
- [x] declared a variable to store users input
- [x] used `std::cin` to prompt user for the number